torchdynamo/eval_frame.py

```diff
@@ -157,6 +159,9 @@ def __call__(self, *args, **kwargs):
 
         @functools.wraps(fn)
         def _fn(*args, **kwargs):
+            if self.first_ctx:
+                log.info("torchdynamo begin tracing")
```

Comment:
It would be great to prefix these milestones with "Step X:", e.g. "Step 1: torchdynamo begin tracing", "Step 2: torchdynamo done tracing", so that user can just search for "Step X+1" to find if/where the next step is run, instead of needing to know the name of next step before they can search for it in the log.